bin/run_docker_dev.sh

```diff
@@ -31,11 +22,7 @@ if [[ $command == "build" ]]; then
     else
         TEST=0
     fi
-    if $PYTHON3; then
-        ARGS="--build-arg PYTHON_BINARY=python3 --build-arg PIP_BINARY=pip3"
-    else
-        ARGS="--build-arg PYTHON_BINARY=python2 --build-arg PIP_BINARY=pip2"
-    fi
+    ARGS=""
```

Comment:
You can safely remove `ARGS` if you also remove the value in

https://github.com/mozilla/wpt-sync/pull/795/files#diff-792e0f8b4e3285a392bb896d8f512eeeR52